TrafficCapture/settings.gradle

```diff
@@ -8,7 +8,11 @@
  */
 
 rootProject.name = 'LoggingProxy'
-include('trafficCaptureProxyServer', 'captureOffloader', 'nettyWireLogging',
+include('captureOffloader',
+        'captureProtobufs',
+        'nettyWireLogging',
+        'trafficCaptureProxyServer',
+        'trafficReplayer',
```

Comment:
Commented on this on your other PR, but I don't understand why `trafficReplayer` is being added to the `LoggingProxy` project.

Reply:
For now, I'm combining them together for convenience.  As a developer, I really need to be able to build "world" with one command.  Putting all of the capture and replay logic into a single hierarchy makes that really simple.  It's a monorepo approach.  I'll (maybe begrudgingly) agree to move off of it when somebody has cycles to invest in another approach that affords me the same abilities.